domenetjenester/kobling/src/main/java/no/nav/foreldrepenger/abakus/kobling/repository/KoblingRepository.java

```diff
@@ -46,7 +46,7 @@ public KoblingRepository(EntityManager entityManager) {
     }
 
     public Optional<Kobling> hentForKoblingReferanse(KoblingReferanse referanse, boolean taSkriveLås) {
-        TypedQuery<Kobling> query = entityManager.createQuery("FROM Kobling k WHERE koblingReferanse = :referanse", Kobling.class);
+        TypedQuery<Kobling> query = entityManager.createQuery("FROM Kobling k WHERE koblingReferanse = :referanse and aktiv=true", Kobling.class);
```

Comment:
kan være litt skummelt med sjekk på aktiv i spørringen. mulig det heller burde gå som en sjekk og en exception dersom ikke aktiv før metoden returnerer?

Reply:
Hm, manglende aktiv var kanskje grunnen til StaleObject. Skulle vi hatt en egen spørring for låsing?